src/plugins/visualizations/public/legacy/build_pipeline.ts

```diff
@@ -95,7 +96,7 @@ const getSchemas = (
     if (isDateHistogramBucketAggConfig(agg)) {
       agg.params.timeRange = timeRange;
       const bounds = agg.params.timeRange ? timefilter.calculateBounds(agg.params.timeRange) : null;
-      agg.buckets.setBounds(agg.fieldIsTimeField() && bounds);
+      agg.buckets.setBounds(agg.fieldIsTimeField() ? (bounds as Bounds) : undefined);
```

Comment:
=( here we cast `TimeRangeBounds` to `Bounds`. Honestly not sure that we should do it 
what about to extend `TimeRangeBounds` from `Bounds`? 

`interface TimeRangeBounds extends Bounds { ... }`

Reply:
now I use TimeRangeBounds  instead of Bounds